modules/tracking/src/precomp.hpp

```diff
@@ -44,6 +44,6 @@
 
 #include "opencv2/tracking.hpp"
 #include "opencv2/core/utility.hpp"
-#include "opencv2/core/private.hpp"
+//#include "opencv2/core/private.hpp"
```

Comment:
If it is not needed, then it is not needed. I don't really see the point of having dead code (especially with includes).